src/platform_impl/windows/dark_mode.rs

```diff
@@ -0,0 +1,222 @@
+/// This is a simple implementation of support for Windows Dark Mode,
+/// which is inspired by the solution in https://github.com/ysc3839/win32-darkmode
+use std::ffi::OsStr;
+use std::os::windows::ffi::OsStrExt;
+
+use winapi::{
+    shared::{
+        basetsd::SIZE_T,
+        minwindef::{BOOL, DWORD, UINT, ULONG, WORD},
+        ntdef::{LPSTR, PVOID, WCHAR},
+        windef::HWND,
+    },
+    um::{libloaderapi, uxtheme, winuser},
+};
+
+lazy_static! {
+    static ref WIN10_BUILD_VERSION: Option<DWORD> = {
+        // FIXME: RtlGetVersion is a documented windows API,
+        // should be part of winapi!
+
+        #[allow(non_snake_case)]
+        #[repr(C)]
+        struct OSVERSIONINFOW {
+            dwOSVersionInfoSize: ULONG,
+            dwMajorVersion: ULONG,
+            dwMinorVersion: ULONG,
+            dwBuildNumber: ULONG,
+            dwPlatformId: ULONG,
+            szCSDVersion: [WCHAR; 128],
+        }
+
+        type RtlGetVersion = unsafe extern "system" fn (*mut OSVERSIONINFOW);
```

Comment:
`RtlGetVersion` has a return type, it's supposed to be `winapi::shared::ntdef::NTSTATUS`.